mindocr/data/builder.py

```diff
@@ -117,13 +120,22 @@ def build_dataset(
     ds = ms.dataset.GeneratorDataset(
                     dataset,
                     column_names=dataset_column_names,
-                    num_parallel_workers=num_workers,
+                    num_parallel_workers=1,
                     num_shards=num_shards,
                     shard_id=shard_id,
-                    python_multiprocessing=True, # keep True to improve performace for heavy computation.
+                    python_multiprocessing=False,
                     max_rowsize =max_rowsize,
                     shuffle=loader_config['shuffle'],
                     )
+    
+    label_encode = RecCTCLabelEncodeV2(24, lower=True)
+    image_resize = RecResizeImgV2([32, 100], infer_mode=is_train, padding=False)
+
+    ds = ds.map(operations=Decode(), input_columns=["image"], num_parallel_workers=1)
+    ds = ds.map(operations=label_encode, input_columns=label_encode.input_columns, output_columns=label_encode.output_columns, num_parallel_workers=1, python_multiprocessing=True)
```

Comment:
Why num_parallel_workers is 1 while multiprocessing is enabled? Seems conflicting.

Reply:
yes. Indeed there is no speed difference when `python_multiprocessing=False`. Can be set to False in this case.